packages/main/src/plugin/extension-loader.ts

```diff
@@ -673,8 +680,18 @@ export class ExtensionLoader {
     } catch (error) {
       console.error('error while reloading extension', error);
     } finally {
-      this.reloadInProgressExtensions.set(extension.id, false);
+      this.reloadInProgressExtensions.delete(extension.id);
     }
+
+    const notification = this.notificationRegistry.addNotification({
+      extensionId: extension.id,
+      type: 'info',
+      title: `Extension ${extension.manifest.displayName} has been updated`,
+    });
```

Comment:
![image](https://github.com/user-attachments/assets/592aadda-9385-4be2-a6e4-d7f34d0afe5a)

small issue: I don't want to have a notification every time I change a file :( (I know I can just disable the extension to avoid having it)

Reply:
I think default is for newcomers. I develop my first extension, I would like to see when it's being refreshed ?

Reply:
I don't know, I feel like this is a bit too much, and in comparison to other tool/framework you don't get notifications.

- When podman-desktop refresh the ui, it does not send a notification :/
- When my spring boot application refresh it does not send a notification
- When my python flask server refresh itself, it does not send a notification.

I am not used to see a notification, and I would expect to be able to disable them at least

Reply:
in the first case, you see traces in the console and if you change on the backend it refreshes.

The last two sample you're referring, it looks like a biased sample or a wrong comparison. You're comparing a Desktop app and a CLI stuff. spring-boot or python app are server-side/cli launch and there is no Desktop/GUI integration but you see something like a trace. If I save my quarkus app I'll see in how many ms the app has been restarted.